lips/lip-7.md

```diff
@@ -0,0 +1,53 @@
+
+The facilitator is responsible for promoting and explaining the proposed TWG to Libra members and must lead it through the voting process by the Technical Steering Committee (TSC). They are also responsible for recruiting other members to the TWG as needed on an ongoing basis, and must help find a replacement facilitator if they must cease work on the initiative.
+
+Once the TWG proposal is presented to the TSC, a simple majority vote in favor forms the TWG. The TWG can then proceed in its stated work and make decisions as needed in the areas covered by its focus.
+
+## Decision Making in a TWG
+The TWG facilitator owns the agenda, meetings, and organization of the TWG, but is not a final decision maker. They are there to drive and facilitate discussion, and help allocate resources so that recommendations, reports, or draft LIPs can be created. They are also responsible for providing a bi-weekly update to the association on the progress the TWG is making and any associated documentation being produced. Bi-weekly updates should be posted in the Libra portal and distributed via email/slack to all Association members.
+
+If there are material changes to the TWG’s scope, timeline, or expected output, a revised proposal must be submitted to the TSC for re-approval.
+
+Any TWG member can propose any topic or idea within the scope of the TWG up for discussion with the group. These discussions should be documented, and as the group comes to agreement on how to proceed, there should be clear ownership among TWG members for drafting, editing, obtaining feedback, and finalizing the group’s working product. There does not need to be total agreement in the TWG on the final output, but a supermajority (67%+) vote in favor is required before proceeding.
+
+Once the TWGs final output is ready and the group votes to proceed, it must be shared with the TSC for review and edits. After the editing process, the TSC must approve the output before it can be circulated within the Association.
+
+If the output is a LIP, the TWG will submit the draft LIP as a pull-request to the libra/lips folder, entering the official [LIP process](https://lip.libra.org/overview/). At this point the LIP gets a LIP Manager to navigate the process from becoming a Draft LIP to a committed LIP in Github. 
+
+In addition to assisting with the LIP process, a LIP Manager can also dedicate engineering resources to develop an implementation of the LIP if one does not exist already. This may mean: 
+Libra Core changes (via github issues and PRs)
+Move Framework changes (same implementation as above)
+Non-blockchain changes in which case things like off-chain work may be affected
+
+After a LIP has sufficient engineering work or a full implementation ready, it enters the last call period. Before it can become Accepted, the TSC must vote on the LIP. A simple majority moves the LIP to Accepted, at which point it becomes up to the maintainers to implement it.
```

Comment:
> at which point it becomes up to the maintainers to implement it

I'm not sure that it becomes up to the maintainers to implement it.  Maintainers may not have the bandwidth to fully implement it and may require additional resources from other Association members